pupil-spa/src/polyfills.ts

```diff
@@ -11,35 +11,19 @@
  * automatically update themselves. This includes Safari >= 10, Chrome >= 55 (including Opera),
  * Edge >= 13 on the desktop, and iOS 10 and Chrome on mobile.
  *
- * Learn more in https://angular.io/docs/ts/latest/guide/browser-support.html
+ * Learn more in https://angular.io/guide/browser-support
  */
 
 /***************************************************************************************************
- * BROWSER POLYFILLS
- */
+* BROWSER POLYFILLS
+*/
 
-/** IE9, IE10 and IE11 requires all of the following polyfills. **/
```

Comment:
Are these polyfill imports:
a) not used on our codebase
b) not required as angular 8 is exposing these in a different way
c)any other reason that is not obvious to me

Reply:
Angular 8 uses differential loading - see the `browserlist` file to see what browsers we are targetting.  The polyfills will be created to support these browsers as required.

Reply:
ok good to know.